src/libutil/util.hh

```diff
@@ -552,7 +552,22 @@ constexpr char treeNull[] = "    ";
 
 /* Determine whether ANSI escape sequences are appropriate for the
    present output. */
-bool shouldANSI();
+typedef enum {
+    colorDefault,
+    colorAlways,
+    colorAuto,
+    colorNever,
+} ColorMode;
```

Comment:
This enum should be a setting. That way it gets command line flags automatically, and can also be set in `nix.conf`. See `Settings::sandboxMode` for an example of an enum setting.